skills/issue-creation/SKILL.md

```
@@ -86,26 +86,31 @@ Intentionally public identifiers are NOT redacted: tool names (`gentle-ai`, `eng
 
 `CLI (commands, flags)` · `TUI (terminal UI)` · `Installation Pipeline` · `Agent Detection` · `System Detection` · `Catalog/Steps` · `Documentation` · `Other`
 
+### Recommended Web Form
+
+Use the web form when the template's auto-labels matter:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)